bundle/edu.gemini.pot/src/main/java/edu/gemini/spModel/gemini/gems/Canopus.java

```diff
@@ -320,24 +306,12 @@ public boolean inRange(ObsContext ctx, Offset offset){
     // Probe arm end, as distance in arcsec from the CWFS guide star
     private static final double PROBE_ARM_END = 8.9;
 
-    // OT-12: Use offset (in arcsec) when instrument is GSAOI
-    private static double getOffsetArcsec(ObsContext ctx) {
-        //     REL-1042: Update the Canopus probe limits in the OT
-//        if (ctx.getInstrument() instanceof Gsaoi) {
-//            return GsaoiDetectorArray.ODGW_HOTSPOT_OFFSET;
-//        }
-        return 0.;
-    }
 
     /**
      * Returns a Shape that defines the AO port in arcsecs, centered at
      * <code>(0,0)</code>.
      */
-    public Shape shape(ObsContext ctx) {
-        double offset = getOffsetArcsec(ctx);
-        return new Ellipse2D.Double(AO_PORT.getX()+offset, AO_PORT.getY()+offset,
-                AO_PORT.getWidth(), AO_PORT.getHeight());
-    }
+    private static final Shape AO_PORT_SHAPE = new Ellipse2D.Double(AO_PORT.getX(), AO_PORT.getY(), AO_PORT.getWidth(), AO_PORT.getHeight());
```

Comment:
I see that the offset arcsec was fixed at 0., but that kind of surprises me.  I wonder what happened to the offset of REL-1042.  It was just commented out without explanation, at least in the code.  At any rate, I suppose we can always add an offset back if necessary and it's good to get rid of unused junk so I support this change.